### ISSUE TITLE
Bring some fresh love here

### ISSUE THREAD

**Issue by amureki** [Thermondo/closeio#93]
We might want to make some cleanups here:

- [x] investigate and fix failing test builds (https://github.com/Thermondo/closeio/pull/96)
- [x] update test packages (pre-commit, pydocstyle, pycodestyle, etc)
- [ ] update README and CONTRIBUTING
- [ ] proper changelog (?)
- [x] fix dependencies versions (?)
- [ ] fix `get_leads` test (or fixture) - causes most problems on closeio while running in parallel travis workers.

**Comment by amureki** [comment]
cc @codingjoe

**Comment by amureki** [comment]
No longer needed, since we are deprecating this package. ;-)
https://github.com/Thermondo/closeio/pull/141
